app/email/controllers.py

```diff
@@ -0,0 +1,57 @@
+"""
+filename: controllers.py
+description: Controllers for email.
+created by: Omar De La Hoz (oed7416@rit.edu)
+created on: 09/25/18
+"""
+
+from email.mime.multipart import MIMEMultipart
+from email.mime.image import MIMEImage
+from email.mime.text import MIMEText
+from email.utils import formataddr
+from app.email.models import huey
+from app import app
+import smtplib
+import config
+
+##
+## @brief      Sends an email.
+##
+## @param      msg   The message object containing:
+## 					 - subtype: Message type (related)
+## 					 - title: Email subject
+## 					 - sender: Tuple with (name, email)
+## 					 - recipients: Array of emails.
+## 					 - html: Rendered html template in string.
+##
+## @return     None
+##
+@huey.task()
+def send_email(msg):
```

Comment:
You should implement retrying the task some amount of times with a backoff on failure.